frontend/src/views/c-authorship.vue

```diff
@@ -359,10 +359,15 @@ export default {
     },
 
     toggleFileActiveProperty(file) {
+      this.scrollFileIntoView(file);
```

Comment:
UX wise, would it be better to only scroll when the element is at the top (like `getBoundingClientRect().y <= 0` or something similar)?
I think github does it this way and it'll also be similar to the original version when the user is not currently in the middle of a file.

Reply:
Yes, that would be better! Have made the necessary modifications